assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -117,3 +121,18 @@ che.keycloak.github.endpoint=NULL
 
 #     The number of seconds to tolerate for clock skew when verifying exp or nbf claims.
 che.keycloak.allowed_clock_skew_sec=3
+
+#     Use the OIDC optional `nonce` feature to increase security.
+che.keycloak.use_nonce=true
```

Comment:
Hello @davidfestal , it seems it's missing properties in che.env

Reply:
wdym ? I see them in `che.env` [here](https://github.com/eclipse/che/pull/8650/files#diff-e4ebb68a2356f42877c3d809f0e569adR546)